src/shared/Internal.Shared.Tasks/Internal.Shared.Tasks.proj

```diff
@@ -0,0 +1,43 @@
+﻿<Project>
```

Comment:
Talked offline and I now get what's going on. Creating two assemblies / packages with identical content, including consolidating the project file elements.

Recommend a true Shared Project instead. Then, split the project information back into the Build.Tasks and Sdk.Tasks, where we expect to find everything.